<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FSettingsDialog</class>
 <widget class="QDialog" name="FSettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="settingPages">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="General">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="2" column="0">
              <widget class="Line" name="line">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pb_ScanNow">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string>Scan now</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="cb_ScanLibOnStartup">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string>Scan for new games on startup</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="pb_gen_ScanForUpdates">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>soon.</string>
               </property>
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string>Scan Now</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <spacer name="verticalSpacer_2">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="10" column="0">
              <widget class="Line" name="line_3">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QCheckBox" name="cb_gen_useTrayIcon">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string>Use Trayicon</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="cb_gen_ScanForUpdates">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string>Scan for Updates on Startup</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QCheckBox" name="cb_gen_StartWithSystem">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string>Start with System</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="lbl_gen_Version">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="text">
                <string>CurrentVersion</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="Line" name="line_2">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QCheckBox" name="cb_gen_minimizeToTray">
               <property name="text">
                <string>Minimize to Tray</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Database">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Interface">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <item row="7" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Standart fontsize</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Load stylesheet</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="pb_ResetStylesheet">
               <property name="text">
                <string>Reset stylesheet</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QComboBox" name="cb_int_language"/>
             </item>
             <item row="9" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QLineEdit" name="le_Stylesheet">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Laguage:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="Line" name="line_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QPushButton" name="pb_selectStylesheet">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>( Requires restart )</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="3">
              <widget class="Line" name="line_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Header fontsize</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QSpinBox" name="sb_int_standardFontsize"/>
             </item>
             <item row="8" column="1">
              <widget class="QSpinBox" name="sb_int_headerFontsize"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Artwork">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="cb_Artwork_UseCache">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cache artwork</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Artwork_ClearCache">
            <property name="text">
             <string>Clear cache</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Artwork_openCache">
            <property name="text">
             <string>Open cache folder</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Artwork_DownloadAll">
            <property name="text">
             <string>Download all available artwork</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="la_Artwork_DownloadStatus">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="WatchedFolders">
         <layout class="QFormLayout" name="formLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QListWidget" name="lw_Folder_FolderList"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_Folder_Add">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_Folder_Remove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="cb_Folder_ForLauncher">
            <property name="text">
             <string>Contains launchable ROMs</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QComboBox" name="cb_Folder_LauncherList">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Launchers">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="1" colspan="3">
           <widget class="QLineEdit" name="le_launcher_suffix"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Compatible files:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="le_launcher_nameEdit"/>
          </item>
          <item row="6" column="3">
           <widget class="QPushButton" name="btn_launcher_remove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="fileLabel">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QListWidget" name="lw_launcher_launchers"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="argLabel">
            <property name="text">
             <string>Arguments</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="3">
           <widget class="QLineEdit" name="le_launcher_argEdit"/>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="btn_launcher_Add">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="btn_launcher_browsePath">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="le_launcher_pathEdit"/>
          </item>
          <item row="5" column="0" colspan="4">
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string>Possible arguments:
$GAMENAME	The game name
$GAMEPATH	The path the game is launched from
$GAMEEXE	The path to the executable, relative to $GAMEPATH</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Logfiles">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QListWidget" name="lw_log_logfiles"/>
          </item>
          <item row="4" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pb_log_clear">
            <property name="text">
             <string>Delete Logfiles</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pb_log_openFolder">
            <property name="text">
             <string>Open Folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="gameSync">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="1">
           <widget class="QSpinBox" name="sb_sync_backups"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="le_sync_path"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Target-Dir</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pb_sync_FolderDialog">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Local saved Backups:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QPlainTextEdit" name="plainTextEdit_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string>To enable Syncing, select the Savegame-Folder in the Settings Dialog in each game, and a synced Directory here - for Example an Dropbox or OwnCloud folder.

Fusion will then keep these Directorys up to date by overriding the old data.

You can specify how many Backups you want to keep locally saved.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
